host/src/server/api/v1/mod.rs

```diff
@@ -58,6 +59,24 @@ pub struct ProofResponse {
     quote: Option<String>,
 }
 
+impl IntoResponse for ProofResponse {
+    fn into_response(self) -> axum::response::Response {
+        axum::Json(serde_json::json!({
+            "status": "ok",
+            "data": self
+        }))
+        .into_response()
+    }
+}
+
+#[derive(Debug, Serialize, ToSchema)]
+#[serde(tag = "status", rename_all = "lowercase")]
+#[allow(dead_code)]
+pub enum Status {
+    Ok { data: ProofResponse },
+    Error { error: String, message: String },
```

Comment:
is {error: String} enough @YoGhurt111 ? Looks error & message are dup

Reply:
The `error` key is the type of error (`Io`, `Db`...) where message is details of that particular instance of the error (e.g. why `Io` failed or why `Db` failed)